.ci/monolithic-local.sh

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env bash

```

Comment:
This can be dropped. `.ci/monolithic-linux.sh` works reasonably well locally, assuming you have already compute the expected projects/runtimes. Running the functions manually also isn't that difficult.

Reply:
I added this file accidentally, it contained the modifications I needed to run. monolithic-linux.sh does not work for anyone who doesn't use Google's modified version of Clang (because of `-gmlt`).

Reply:
Makes sense.

Although `-gmlt` is very much not a google-specific extension. These containers run the latest release clang built from source straight from the release branch. The option is documented in the clang options manual. https://clang.llvm.org/docs/ClangCommandLineReference.html#cmdoption-clang-gline-tables-only